cmd/test.go

```diff
@@ -75,13 +76,57 @@ func (t *Test) getTestConfig(path *string, proxyPort *uint32, appCmd *string, te
 	if *apiTimeout == 5 {
 		*apiTimeout = confTest.ApiTimeout
 	}
-	noiseJSON, err := test.UnmarshallJson(confTest.Noise, t.logger)
+	noiseJSON, err := test.UnmarshallJson(confTest.GlobalNoise, t.logger)
 	if err != nil {
-		t.logger.Error("Failed to unmarshall the noise flag", zap.Error((err)))
+		t.logger.Error("Failed to unmarshall the noise flag")
+		return err
 	}
-	*noiseConfig = map[string]interface{}{}
-	(*noiseConfig)["body"] = noiseJSON.(map[string]interface{})["body"]
-	(*noiseConfig)["header"] = noiseJSON.(map[string]interface{})["header"]
+
+	globalScopeVal := noiseJSON.(map[string]interface{})["global"]
+	
+	bodyOrHeaderVal := globalScopeVal.(map[string]interface{})
+
+	(*globalNoise)["body"] = map[string][]string{}
+	for field, regexArr := range bodyOrHeaderVal["body"].(map[string]interface{}) {
+		(*globalNoise)["body"][field] = []string{}
```

Comment:
Lets say I gave this noise :

```
"noise":
    { 
      "global":
         {
            "body":
              {  "cart" : 
                     {  "item" :
                           "id" : "someregex"
                     }
                }
          }
}
```

will this work I want noise out id in this case...? I think the for loop is gonna fail here `regexArr.([]interface{})`

Reply:
You have to provide it like:
```go
"noise":
    { 
      "global":
         {
            "body":
              {  
                   cart.item.id : ["regex"]
              }
          }
    }
```
The jsonmatch() would be expecting input in this dotted format.

Reply:
Got it! You work on other comments and readme.